docs/conventions.md

```diff
@@ -66,10 +66,14 @@ A placeholder must meet the following conditions:
 ## Console command
 A console command must meet the following conditions:
 
-- It shouldn't have any business logic.
-- Only basic input validation and output formatting should be present in the console command.
-- `protected static $defaultName` should not be used for performance reasons and future deprecation in Symfony version 6.1.
-- `protected const NAME` should be provided and passed to the parent constructor as a parameter instead of `protected static $defaultName`.
+- Console command __SHOULD__ have no business logic.
```

Comment:
I'm not sure we should overwrite console command conventions, as it's already edited by a technical writer.